plugins/quota/actor.go

```diff
@@ -0,0 +1,96 @@
+package quota
+
+import (
+	"os"
+	"path"
+
+	"github.com/gluster/glusterd2/glusterd2/daemon"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/glusterd2/xlator"
+	"github.com/gluster/glusterd2/pkg/errors"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type quotadActor struct{}
+
+const (
+	quotaDaemonKey = "enable"
+)
+
+// isQuotadAffectedNode returns true if there are local bricks of volume on
```

Comment:
The comment needs to be updated. Here's a suggested text:

```
isQuotadStopRequired checks if the quota daemon has to be stopped.
The quotad process needs to be stopped on a peer only when that peer
does not have bricks belonging to a quota enabled volume.
```

Reply:
make better sense